server/config/routes.js

```diff
@@ -366,6 +368,7 @@ exports.configure = function configure(app) {
   app.get('/reports/archive/:uuid', report.sendArchived);
   app.delete('/reports/archive/:uuid', report.deleteArchived);
 
+  app.get('/debtorAccountBalence/:fiscalYearId', reportDebtor.debtorAccountBalance);
```

Comment:
This url should read `debtorAccountBalance`.

Almost all of our reports use the url `reports/....`.  Could you use that style of URL?  It will make things easier to debug when bugs occur.